models/stg_hubspot__contact.sql

```diff
@@ -10,7 +10,7 @@ with base as (
 
     select
         id as contact_id,
-        {{ fivetran_utils.remove_prefix_from_columns(columns=columns, prefix='property_', exclude=['id']) }}
+        {{ fivetran_utils.remove_prefix_from_columns(columns=columns, prefix='property_', exclude=['id', 'contact_id']) }}
```

Comment:
is `contact_id` supposed to be `property_contact_id`? based on https://github.com/fivetran/dbt_hubspot_source/pull/9/files